src/github.com/getlantern/flashlight/app/app.go

```diff
@@ -52,20 +53,11 @@ func (app *App) Init() {
 
 // LogPanicAndExit logs a panic and then exits the application.
 func (app *App) LogPanicAndExit(msg string) {
-	cfg, err := config.Init(settings,
-		flashlight.PackageVersion,
-		app.Flags["configdir"].(string),
-		app.Flags["stickyconfig"].(bool),
-		app.Flags,
-	)
-	if err != nil {
-		panic("Error initializing config")
-	}
 	if err := logging.EnableFileLogging(); err != nil {
 		panic("Error initializing logging")
 	}
 
-	<-logging.Configure("", cfg.Client.DeviceID, flashlight.Version, flashlight.RevisionDate)
+	<-logging.Configure("", "dummy-device-id-for-panic", flashlight.Version, flashlight.RevisionDate)
```

Comment:
We don't need the device ID when reporting a panic, and the previous code went out of its way to get a config just for that purpose.